src/utils.ts

```diff
@@ -629,6 +629,22 @@ export function reportUrlType(url: string, telemetryReporter: Readonly<Telemetry
     telemetryReporter.sendTelemetryEvent('user/browserNavigation', { 'urlType': urlType });
 }
 
+export async function reportFileExtensionTypes(telemetryReporter: Readonly<TelemetryReporter>): Promise<void> {
+    const files = await vscode.workspace.findFiles('**/*.*', '**/node_modules/**');
+    const extensionSet: Set<string> = new Set();
+    for (const file of files) {
+        const extension: string | undefined = file.path.split('.').pop();
+        if (extension) {
+            extensionSet.add(extension);
+        }
+    }
+    let fileTypes = '';
+    for (const extension of extensionSet.keys()) {
+        fileTypes += extension + ';';
```

Comment:
Right now, the property value is set as `"css;js;html;json;"` since the property value must be a string. We could search for `".js;"` in the property value in Kusto later to check if a given file extension type exists. If there is a better way to organize this information, I am open to changing this as well.

Reply:
Hmm, I think a better approach might be using the "measurements" field and using counts or only adding properties we care about.
Something like
``` 
sendTelemetryEvent('workspace/filetypes', undefined, {
   css: 1,
js: 1,
html: 1,
jsx: 0,
...
});
```

Reply:
I think this might be easier to parse from kusto

Reply:
Hmm yeah I was considering this approach.  Since we don't have an exact list of file extensions we care about, I was thinking that the object would only have existing file extensions.  In that case everything would just be set to 1 which I thought was kind of redundant.  

For example:
`{js: 1, jsx: 1, css: 1}` or `{css: 1, js: 1, json: 1}`

I'm going to run with your idea and maybe change the value to account for number of files with each file extension such as:

`{js: 2, css: 1, json: 1, total: 4}`

Reply:
Sounds good to me!

Reply:
New format: `workspace/metadata: undefined, {"css":1,"js":1,"html":1,"json":2,"total":5}`